ntbs-service/wwwroot/css/notificationBanner.scss

```diff
@@ -0,0 +1,83 @@
+@import "./_colors.scss";
+
+.notification-banner-head-container {

```

Comment:
Last small niggle - I'm not sure the word "container" on these classes is really adding that much. It's a bit of a generic-css word (like "service" classes ;) ) and in this case isn't really differentiating between the container and the contained thing. Should we drop it form here (and the other classes below)?